src/main/java/be/ugent/IfcSpfReader.java

```diff
@@ -301,6 +309,9 @@ public void convert(String ifcFile, String outputFile, String baseURI) throws IO
 		HttpOp.setDefaultHttpClient(HttpClientBuilder.create().useSystemProperties().build());
 		om = ModelFactory.createOntologyModel(OntModelSpec.OWL_DL_MEM_TRANS_INF);
 		in = IfcSpfReader.class.getResourceAsStream("/" + exp + ".ttl");
+		if (in == null)
```

Comment:
Formatting need corrected.

Reply:
Formatting was corrected using IFC-to-RDF Convertor Eclipse Formatter